ledger-core/application/benchs/api_utils_test.go

```diff
@@ -0,0 +1,222 @@
+// Copyright 2020 Insolar Network Ltd.
+// All rights reserved.
+// This material is licensed under the Insolar License version 1.0,
+// available at https://github.com/insolar/assured-ledger/blob/master/LICENSE.md.
+
+package benchs
```

Comment:
We can't use common API for functests and that?
Why API doesn't use ctx for cancelling requests?

Reply:
I was laze and just picked api helpers from functests. Maybe will rework this part later

Reply:
functests API is not designed well and I don't wan t to start reworking it. (it uses only localhost and defaultPorts instead of dynamic stuff)

The best way to simplify this is move towards openapi (swagger) for both server and client code. Not sure that we need it right now

Reply:
added context propagation